embedchain/loaders/excel_file.py

```diff
@@ -2,10 +2,11 @@
 import importlib.util
 
 try:
+    import unstructured  # noqa: F401
```

Comment:
Do you  need to import unstructured here? Seems like you are not using it explicitly in this file, right? 

Same comment for other related changes.

Reply:
Resolved offline.